### ISSUE TITLE
Problem with --checkout reclone

### ISSUE THREAD

**Issue by audreyr** [audreyr/cookiecutter#546]
The message should ask me about recloning `/Users/audreyr/.cookiecutters/cookiecutter-pypackage`, not `/Users/audreyr/.cookiecutters`.

```
$ cookiecutter https://github.com/eliasdorneles/cookiecutter-pypackage/ -c adding-travis-setup-for-pypi-deployment
You've cloned /Users/audreyr/.cookiecutters before. Is it okay to delete and re-clone it? [yes]: 
```

**Comment by hackebrot** [comment]
It works just fine if the url to the template is as follows (without the ``/`` at the very end) :no_mouth: 

```bash
https://github.com/eliasdorneles/cookiecutter-pypackage
```

**Comment by hackebrot** [comment]
```
(cookiecutter)[raphael@hackebook cookiecutter (fix-re-clone-prompt)]$ cookiecutter https://github.com/eliasdorneles/cookiecutter-pypackage -c adding-travis-setup-for-pypi-deployment
You've cloned /Users/raphael/.cookiecutters/cookiecutter-pypackage before. Is it okay to delete and re-clone it? [yes]: 
Cloning into 'cookiecutter-pypackage'...
remote: Counting objects: 609, done.
remote: Compressing objects: 100% (24/24), done.
remote: Total 609 (delta 10), reused 0 (delta 0), pack-reused 585
Receiving objects: 100% (609/609), 95.98 KiB | 95.00 KiB/s, done.
Resolving deltas: 100% (319/319), done.
Checking connectivity... done.
Branch adding-travis-setup-for-pypi-deployment set up to track remote branch adding-travis-setup-for-pypi-deployment from origin.
Switched to a new branch 'adding-travis-setup-for-pypi-deployment'
full_name [Raphael Pierzina]: 
email [raphael@hackebrot.de]: 
github_username [hackebrot]: 
project_name [Python Boilerplate]: 
repo_name [boilerplate]: 
project_short_description [Python Boilerplate contains all the boilerplate you need to create a Python package.]: 
release_date [2015-01-11]: 
pypi_username [hackebrot]: 
year [2015]: 
version [0.1.0]: 
use_pypi_deployment_with_travis [y]: 
Error: "boilerplate" directory already exists
```

**Comment by hackebrot** [comment]
Hey @audreyr! :wave: 

What's your opinion on this guy? Shall we change anything?

**Comment by audreyr** [comment]
Thanks for figuring this out @hackebrot. Okay, this is a bug, but I'll take off the 1.1.1 milestone since it's known behavior and not absolutely urgent.

**Comment by pydanny** [comment]
Closed thanks to @ramiroluz and #573
